ex1/ex1.js

```diff
@@ -6,3 +6,29 @@
  *     fn(",,,") === 2
  *     fn("hel,lowo,rld") === -1
  */
+
```

Comment:
Please revert all files except ex6. Next time, please create the branch from master (which has none of the new files).
I can explain about this over the phone.